drift_dev/lib/src/analysis/resolver/dart/view.dart

```diff
@@ -209,7 +227,7 @@ class DartViewResolver extends LocalElementResolver<DiscoveredDartView> {
       if (parts.length > 1) {
         final reference =
             references.firstWhereOrNull((ref) => ref.name == parts[0]);
-        if (reference == null) {
+        if (reference == null || reference.table == null) {
```

Comment:
Is this a leftover from a previous iteration? `TableReferenceInDartView.table` is a non-nullable field.

Reply:
Done. Yes not sure what I tried there.